src/sections/Meshery/Meshery-terminal/index.js

```diff
@@ -155,135 +155,55 @@ const MesheryTerminal = () => {
                   {
                     frames: 8,
                     code: [
-                      "Removing installation files and starting Meshery",
-                      "Removing installation files and starting Meshery.",
-                      "Removing installation files and starting Meshery..",
-                      "Removing installation files and starting Meshery...",
-                      "Removing installation files and starting Meshery....",
-                      "Removing installation files and starting Meshery.....",
-                      "Removing installation files and starting Meshery......",
-                      "Removing installation files and starting Meshery.......",
+                      "downloading manifest files from https://github.com/meshery/meshery/tree/v0.5.60/install/deployment_yamls/k8s",
```

Comment:
```suggestion
                      "downloading manifest files from https://github.com/meshery/meshery/tree/v0.6.0/install/deployment_yamls/k8s",
```
Considering the fact we're switching to k8's from v6, let's keep it v0.6.0 here